app/helpers/application_helper.rb

```diff
@@ -252,4 +252,20 @@ def allowed_boards(obj, user)
     authored_ids = BoardAuthor.where(user: user).select(:board_id)
     Board.where(id: obj.board_id).or(Board.where(authors_locked: false)).or(Board.where(id: authored_ids)).ordered
   end
+  def get_owed
+    ids = Post::Author.where(user_id: current_user.id, can_owe: true).group(:post_id).pluck(:post_id)
+    @posts = Post.where(id: ids)
+    drafts = ReplyDraft.where(post_id: @posts.select(:id)).where(user: current_user).pluck(:post_id)
+    solo = Post::Author.where(post_id: ids).group(:post_id).having('count(post_id) < 2').pluck(:post_id)
+    @posts = @posts.where.not(last_user: current_user).or(@posts.where(id: (drafts + solo).uniq))
+    @posts = @posts.where.not(status: [:complete, :abandoned])
+    hiatused = @posts.hiatus.or(@posts.where('tagged_at < ?', 1.month.ago))
+
+    if current_user.hide_hiatused_tags_owed?
+      @posts = @posts.where.not(status: :hiatus).where('tagged_at > ?', 1.month.ago)
+      @hiatused_exist = true if hiatused.count > 0
+    end
+
+    return @posts.count
```

Comment:
None of these should be instance variables; among other reasons, we have a number of controllers which set a different @posts variable and they'll conflict. You're not using them anywhere else, so they can safely drop the @.